docs/en/sql-reference/sql-statements/data-definition/ALTER_TABLE.md

```diff
@@ -28,22 +30,23 @@ ALTER TABLE [<db_name>.]<tbl_name>
 alter_clause1[, alter_clause2, ...]
 ```
 
-`alter_clause` is classified into six operations: partition, rollup, schema change, rename, index, swap, comment, and compact.
+`alter_clause` is classified into operations, including rename, comment, partition, bucket, column, rollup index, bitmap index, table property, swap and compaction.
 
 - rename: renames a table, rollup index, or partition. **Note that column names cannot be modified.**
 - comment: modifies the table comment (supported from **v3.1 onwards**).
-- swap: atomic exchange of two tables.
 - partition: modifies partition properties, drops a partition, or adds a partition.
-- schema change: adds, drops, or reorders columns, or modifies column type.
-- rollup: creates or drops a rollup index.
-- index: modifies index (only Bitmap index can be modified).
-- compact: performs manual compaction to merge versions of loaded data (supported from **v3.1 onwards**).
+- bucket: modifies bucketing method and the number of buckets.
```

Comment:
![image](https://github.com/StarRocks/starrocks/assets/57167462/3b5b1c8f-803d-43cf-924d-6d45b5e8d9f1)
-> the bucketing method